README.rst

```diff
@@ -1,5 +1,58 @@
-# barcode
-Simple lightweight python barcode EAN generator. Output format SVG and ASCII
+# barcode-generator (EAN13 compatible)
+Simple lightweight python barcode EAN 13 generator. Output format SVG and ASCII
 
-Usage: 
-python barcode.py
+Only provide EAN 13 barcode type
+
+Summary
+-------
+* Enter your 13 digits
+* Press enter
+* Copy output SVG on html file
+* Open this html file with browser
+* Have Fun !
+
+Install
+-------
+`$ pip install barcode-generator`
+
+or
+
+`$ git clone http://github.com/4383/barcode`
+
+`$ cd barcode`
+
+`$ python setup.py install`
+
+
+Usage examples
+------
+`$ barcode`
+
+`$ barcode 0123456789123`
+
+`$ barcode 0123456789123 --ascii --html --svg`
+
+`$ barcode 0123456789123 --ascii --html --svg -o "these fruits, these vegetables"` 
+
+`$ python barcode.py`
```

Comment:
I think that all the lines below should be a multi-lines code snippet to better render it.

Reply:
I thought this was better because of the descriptions. I can implement the change, and it would be more consistent, but I don't think it would be more readable.

Reply:
I just removed the description, in the end, the options are relatively self-explanatory. and it does look cleaner

Reply:
Ok no problem. Thanks